pkg/rancher-desktop/backend/k3sHelper.ts

```diff
@@ -195,7 +195,7 @@ export default class K3sHelper extends events.EventEmitter {
       for (const versionString of cacheData.versions) {
         const version = semver.parse(versionString);
 
-        if (version) {
+        if (version && version.compare(this.minimumVersion) >= 0) {
```

Comment:
No, I agree. And there are plenty more instances where we call `compare` and immediately compare the return value against an integer constant instead of using one of the specific relation predicates. But I've resisted the temptation to change all of them right now.